webpack.config.js

```diff
@@ -67,7 +67,9 @@ module.exports = {
       //NOTE: Bootstrap css configuration
       {
         test: /\.css$/,
-        include: /node_modules\\bootstrap/,
+        include: [/node_modules\\bootstrap/,
+          /src\\common\\components/,
```

Comment:
You don't need to add src/common/components because next loader is loading .scss files from src